dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dashboard/DashboardUsersService.java

```diff
@@ -0,0 +1,28 @@
+package edu.harvard.iq.dataverse.dashboard;
+
+import edu.harvard.iq.dataverse.DataverseRequestServiceBean;
+import edu.harvard.iq.dataverse.EjbDataverseEngine;
+
+import javax.ejb.Stateless;
+import javax.inject.Inject;
+
+@Stateless
+public class DashboardUsersService {
+
+    private EjbDataverseEngine commandEngine;
+    private DataverseRequestServiceBean dvRequestService;
+
+    // -------------------- CONSTRUCTORS --------------------
+
+    @Deprecated
+    public DashboardUsersService() {
+    }
+
+    @Inject
+    public DashboardUsersService(EjbDataverseEngine commandEngine, DataverseRequestServiceBean dvRequestService) {
+        this.commandEngine = commandEngine;
+        this.dvRequestService = dvRequestService;
+    }
+
+    // -------------------- LOGIC --------------------
```

Comment:
No logic in this class, is it needed?

Reply:
This was changed in another task. I've merged with develop so it up to date now.